### DISCUSSION TITLE
Concurrency group

### DISCUSSION THREAD

**Discussion by gabsmprocha** [community/community#21342] [Actions]
Hi guys!

How can i use concurrency group for two workflows? for example I have a workflow called CD, which has the following concurrency:

`concurrency: cd-${{ github.event.number }}`

And another workflow called CI. I would like the CI to cancel whenever there is another CI from the same group running, or a CD, so I did:

```
concurrency:
    group: ci-${{ github.event.number }} || cd-${{ github.event.number }}
    cancel-in-progress: true
```

but it doesn't work, any idea where I could be going wrong?

**Comment by falbrechtskirchinger** [comment]
You need to set your `concurrency.group`s to the same value. A workflow is canceled when another workflow with the same name is queued up.
